app/javascript/mastodon/components/column_header.js

```diff
@@ -101,6 +113,9 @@ export default class ColumnHeader extends React.PureComponent {
     if (multiColumn && pinned) {
       pinButton = <button key='pin-button' className='text-btn column-header__setting-btn' onClick={onPin}><i className='fa fa fa-times' /> <FormattedMessage id='column_header.unpin' defaultMessage='Unpin' /></button>;
 
+      if(systemHdrPin)
+          pinButtonHdr = <button key='pin-button' className='column-header__button' onClick={onPin}><i className='fa fa fa-times' /></button>;
```

Comment:
please use brackets around your if statements.

Reply:
Thanks for the assisting, I will adjust my files and will turn in a new PR